<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reservasion_add</class>
 <widget class="QDialog" name="reservasion_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;ori1Uni&quot;;</string>
   </property>
   <property name="text">
    <string>렌터카 예약</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="reserv_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>194</width>
     <height>41</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>7</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>반납시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>예약시간</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="return_time">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>194</width>
     <height>41</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>7</month>
     <day>2</day>
    </datetime>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>전체</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>휘발유</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경유</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LPG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>전기</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>621</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>차량 선택</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>168</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="booking">
      <property name="text">
       <string>예약</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>뒤로</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
